README.md

```diff
@@ -155,7 +155,13 @@ Apply `Schemafile`
 No change
 ```
 
-## Limitation on MySQL 8.0
+## Supported versions
+
+- Ruby 3.1 to 3.4
+- ActiveRecord 7.0 to 7.2
+- MySQL 5.6 to 8.0
```

Comment:
How about dropping support for MySQL 5.x here or in another pull request?

Reply:
Official support for MySQL 5.x has already ended:
https://endoflife.date/mysql

That said, some managed services like AWS still offer extended support:
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html

Maybe it would make sense to handle both the end of support for 5.x and the addition of MySQL 8.4 (LTS) in a separate PR.

Reply:
I'll open a separate issue for that.